tests/test_RBAC.py

```diff
@@ -0,0 +1,138 @@
+#
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+import json
+
+import pytest
+from ocean_lib.common.agreements.service_types import ServiceTypes
+from ocean_lib.models.data_token import DataToken
+
+from ocean_provider.constants import BaseURLs
+from ocean_provider.exceptions import RequestNotFound
+from ocean_provider.utils.accounts import generate_auth_token
+from ocean_provider.validation.requests import RBACValidator
+from tests.test_helpers import (
+    mint_tokens_and_wait,
+    get_dataset_ddo_with_access_service,
+    send_order,
+)
+
+
+def test_null_validator():
+    with pytest.raises(RequestNotFound):
+        RBACValidator()
```

Comment:
ah, please implement RBACValidator as an Abstract Base Class just to be sure. Keep the tests, but alter it to take a gibberish request instead.